etc/api_docker-compose.override.yml

```diff
@@ -9,6 +9,9 @@ services:
       # Have the http router redirect to https, the middleware is defined in static.yml
       - "traefik.http.routers.${COMPOSE_PROJECT_NAME}-http.middlewares=redirect-to-https@file"
 
+  mariadb:
```

Comment:
The changes for ARM64 compatibility (I assume we're only talking the switch to mariadb) looks ok. We should probably pin it to a more specific tag though - by looking at https://github.com/itk-dev/devops_itkdev-docker/blob/develop/images/mariadb/Dockerfile it seems 10.5 is the way to go.

Reply:
We can do that, I my thinking was that we wouldn't be able to pin point any version as this will run in a lot of places and probably different hosting setups, so why not run latest in dev?